src/main.ts

```diff
@@ -196,6 +196,11 @@ function createWindow() {
 // IPC
 
 app.on('ready', () => {
+  // @ts-ignore
```

Comment:
I have been struggling to get rid of this. It is here, because TS complains about "`Property 'Locale' does not exist on type 'typeof Intl'`", which makes little sense to me. Everything works fine if I ignore the error, my IDE doesn't complain. I have not been able to find any useful explanation on the web.

On the other hand the `Intl.Locale` is what I've found to reliably extract the language code from the locale code. This all feels clunky and out of place, am I missing something? This should be simple :/.

Nevertheless, works on Ubuntu and Windows, tried English, Polish, German and German-Liechtenstein localized systems, behaves as expected.

Reply:
Can we simplify to `if (['de', 'de-AT', 'de-CH', 'de-DE'].includes(app.getLocale()) { change lang ... }`?

Reply:
Is this an intrinsic v8 type, because I don't see an import anywhere?

Reply:
ok, it turns out always worked with locale sent as language, it only caused a bunch of errors, but worked anyway :shrug:. THere's even a setting which makes these errors go away. It still complains if the language is not supported, but I think this is good enough.

Reply:
Are these errors due to the fact that we assign some unkown language, e.g. Polish, and then i18next complains about missing keys? That's not ideal. I suggest a simple check if the app locale is `de` and switching to it.

Reply:
yyyyeah, it actually errors on each render. I give up, explicit listing of `de` locales it is...